### ISSUE TITLE
Information Refresh

### ISSUE THREAD

**Issue by SkylerDuggan** [tacticaltofu/drivetest-openings#2]
Great job! Is there a way to forcefully refresh the site's information? Currently, it's 5 days old and I was just wondering if I could update it myself.

**Comment by tacticaltofu** [comment]
No, sorry. The original plan was for a script to be updating the site data every hour or so. The problem is that drivetest.ca locks me out if I make requests that frequently. Currently working on another way to scrape the available dates (hopefully without having to use Selenium).

**Comment by SkylerDuggan** [comment]
> No, sorry. The original plan was for a script to be updating the site data every hour or so. The problem is that drivetest.ca locks me out if I make requests that frequently. Currently working on another way to scrape the available dates (hopefully without having to use Selenium).

What's wrong with using Selenium?

**Comment by tacticaltofu** [comment]
> > No, sorry. The original plan was for a script to be updating the site data every hour or so. The problem is that drivetest.ca locks me out if I make requests that frequently. Currently working on another way to scrape the available dates (hopefully without having to use Selenium).
> 
> What's wrong with using Selenium?

Nothing's really wrong with it - it is just pretty slow. The script I've written also does not work in headless mode, so in order to host the script I will need to find a platform that allows me to have a GPU (not many free hosting services offer this).
